pkg/cmd/extension/command.go

```diff
@@ -29,7 +29,9 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 
 			The name of the extension repository must start with "gh-" and it must contain an
 			executable of the same name. All arguments passed to the %[1]sgh <extname>%[1]s invocation
-			will be forwarded to the %[1]sgh-<extname>%[1]s executable of the extension.
+			will be forwarded to the %[1]sgh-<extname>%[1]s executable of the extension. You can also
+			pass the extension repository's organization and name like "owner/gh-extension" to
+			disambiguate extensions with the same <extname>.
```

Comment:
Since it's currently not possible to install two extensions with the same `<extname>`, even if they are owned by different accounts, does it add value to add this feature of passing the full extension name?

Reply:
I actually didn't realize that, or notice when going through the code for this. Makes sense given the current restrictions, but with this new feature - given a full extension name - should we allow installing duplicate short names? In the fallback code we could throw an actionable error, much like the "command not found, did you mean ..." error.

Not sure if there are any dups today (sadly, topics are not obviously searchable), but there are many extensions and I'd be surprised if too many people thought about this when they created their extension, not to mention there's no good UX for renaming them later.

Reply:
> should we allow installing duplicate short names? In the fallback code we could throw an actionable error, much like the "command not found, did you mean ..." error.

Sure, your feature of specifying full names would make it possible to disambiguate which extension exactly you want to run, but for now I don't think duplicate extension names would result in a great experience for the user. I would prefer for users to be always able to invoke an extension by its short name, e.g. `gh label`.